notebooks/00_quick_start/run_notebook_on_azureml.ipynb

```diff
@@ -0,0 +1,296 @@
+    "shutil.copy(NOTEBOOK_NAME, notebook_path)\n",
+    "        \n",
+    "exp = Experiment(workspace=ws, name=EXPERIMENT_NAME)\n",
+    "\n",
+    "run_config_aml = RunConfiguration()\n",
+    "run_config_aml.target = \"amlcompute\" # possible to use different compute targets\n",
+    "run_config_aml.amlcompute.vm_size = 'STANDARD_D2_V2'\n",
+    "run_config_aml.environment.docker.enabled = True\n",
+    "run_config_aml.environment.docker.base_image = DEFAULT_CPU_IMAGE\n",
+    "run_config_aml.environment.python.user_managed_dependencies = False\n",
+    "run_config_aml.auto_prepare_environment = True\n",
+    "run_config_aml.environment.python.conda_dependencies = CondaDependencies(conda_dependencies_file_path='reco_base.yaml')\n",
+    "\n",
+    "cfg = NotebookRunConfig(source_directory='../../',\n",
+    "                            notebook='notebooks/00_quick_start/' + NOTEBOOK_NAME,\n",
+    "                            output_notebook='outputs/out.ipynb',\n",
+    "                            parameters={\"MOVIELENS_DATA_SIZE\": \"100k\", \"TOP_K\": 10},\n",
+    "                            run_config=run_config_aml)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`exp.submit()` will submit source_directory folder and designate notebook to run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run = exp.submit(cfg)\n",
+    "run"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Above cell should output similar table as below\n",
+    "![Experiment submit output](https://recodatasets.blob.core.windows.net/images/aml_papermill_cell_output.jpg)\n",
+    "After clicking \"Link to Azure Portal\", experiment run details tab looks like this\n",
+    "![Azure Portal Experiment](https://recodatasets.blob.core.windows.net/images/aml_papermill_portal.jpg)\n",
+    "Experiment output is out.ipynb. You can download and view this file locally.\n",
+    "![Output file](https://recodatasets.blob.core.windows.net/images/aml_papermill_outputs_tab.jpg)\n",
+    "#### Jupyter widget\n",
+    "\n",
+    "Jupyter widget can watch the progress of the run.  Like the run submission, the widget is asynchronous and provides live updates every 10-15 seconds until the job completes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "4c2b25a2ebad45428fe2b0613080ff70",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "_UserRunWidget(widget_settings={'childWidgetDisplay': 'popup', 'send_telemetry': False, 'log_level': 'INFO', '…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "RunDetails(run).show()"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
I'm not aware of a way to do so, because the output from this run is a file 'out.ipynb' where user can download and see the output cell value. If the original notebook (sar_movielens) uses run.log() then we could do the same. I heard there're work items in backlog to support such logging. Adding @heatherbshapiro since she probably knows more on this topic.

Reply:
This is key. There has to be a way for the user to see the results, what is the benefit of running a notebook remotely if I can't see what is the output of the notebook?

Reply:
Yes you can do 
```
metrics = run.get_metrics()
print(metrics)
```

Reply:
Yup this works